### PR TITLE
Fix missing git depenendency for Jupyter docker image.

### PR DESCRIPTION
**Pull request by traviscollins**
The jupyter notebook docker image includes a tutorial named "overfit and underfit" that has a step that requires git to run. The nightly-py3-jupyter image does not include git in the docker image. This is my attempt to add git to that image.

### CODE REVIEW

**Comment by traviscollins** [comment]
@googlebot I signed it!

**Comment by gbaned** [comment]
@traviscollins thank you for your contribution, please sign CLA.

**Comment by traviscollins** [comment]
I tried but after logging into my google account through the CLA process, I get a 500 error from the google site.

**Comment by gbaned** [comment]
@traviscollins Please try again and keep us posted. Because this cannot be proceed until CLA reflects YES. Thanks!

**Comment by gbaned** [comment]
@traviscollins gentle ping to sign CLA and keep us posted. Thanks!

**Comment by traviscollins** [comment]
There’s nothing I can do here. The CLA site returns a 500 error and no useful explanation. You’ll have to contact the CLA group to fix the issue, accept this issue (I consent to the terms), or close the issue.

**Comment by traviscollins** [comment]
@googlebot I signed it!

**Comment by gbaned** [comment]
@traviscollins Can you please resolve conflicts? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by angerson** [comment]
Thanks for this, sorry about the trouble with the CLA. Another PR already added git to the images; I've verified that `docker run -it --rm tensorflow/tensorflow:nightly-jupyter git version` works, so I'll close this PR.
